0014/README.md

```diff
@@ -0,0 +1,181 @@
+---
+name: Time Series Receiver Protocol
+shortname: 16/TSRP
+status: draft
+editor: Andrew Kokhanovskyi <ak@kaaiot.io>
+---
+
+<!-- toc -->
+
+
+# Introduction
+
+Time Series Receiver Protocol (TSRP) is designed to communicate endpoint data samples from transmitter to receiver services in the Kaa platform.
```

Comment:
maybe "data points"? Looks like a copycat from 13/DRP

Reply:
Copycat it is :)